config/crds/migration_v1alpha1_migplan.yaml

```diff
@@ -66,6 +66,10 @@ spec:
                 - serviceAccount
                 type: object
               type: array
+            indirectImageMigration:
```

Comment:
Yes, this was a deliberate choice because we wanted the direct option to be the default -- not specifying these fields results in "indirectFoo=false", meaning "we want direct migration".